setup.cfg

```diff
@@ -23,12 +23,13 @@ install_requires =
     dimod>=0.12.3
     dwave-system>=1.18.0
     numpy>=1.20.0
-    scikit-learn>=1.0.2 
+    scikit-learn>=1.2.1
+    scipy>=1.7.3
 packages = 
     dwave
     dwave.plugins
     dwave.plugins.sklearn
-python_requires = >=3.7
+python_requires = >=3.8
```

Comment:
We should note this in the release notes. E.g. `Drop support for Python 3.7.` or `Upgrade your python to 3.8+. We no longer support python 3.7 and below.`.